jenkins/release-tag/release-tag.jenkinsfile

```diff
@@ -0,0 +1,54 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211130'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(
```

Comment:
As an idea, this job could take a version number without a build ID, because the URL for the manifest for a well-known version should be known. Or it could take a URL to the manifest. This avoids the need to download anything from S3. It's no big deal though.

Reply:
Getting the manifest URL will still need the build ID, otherwise we will need to take the manifest URL as the parameter input. I think essentially both way are the same, good for either way.

Reply:
Also, We are bundling this manifest with the released artifact as well. Does it makes sense to download from publicly downloadable link 
```
Example: https://artifacts.opensearch.org/releases/bundle/opensearch/1.2.4/opensearch-1.2.4-linux-x64.tar.gz
```
We can download from public url , unzip and use the manifest. In that case just the version is required.

Reply:
I liked this idea of using this public link for transparency purpose. Just make sure few things: 
This release artifacts matches the build ID that we will be using for each release right?
Also I actually uses the builds manifest instead of the dist manifest from link like `https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/$version/$build_number/linux/x64/build/opensearch/manifest.yml`; using the dist manifest from the release artifacts will not include some components that won't redistribute anything (e.g. `common-util`). So we will still need to manually tag for them.

Reply:
Makes sense. Thanks!

Reply:
Implemented in https://github.com/opensearch-project/opensearch-build/pull/1705

Reply:
We can use the manifest directly from released artifact now. Would recommend to use that instead of build id. In that case just the version is required to retrieve the manifest.
Thanks!